martus-client/source/org/martus/client/swingui/jfx/landing/general/ManageServerSyncRecordsController.java

```diff
@@ -209,6 +214,16 @@ protected void succeeded()
 
 			serverMyDrafts = getValue();
 		}
+
+		@Override
+		protected void failed()
```

Comment:
The failed methods can be extracted into the main class and and reused by all three tasks.